go-mode.el

```diff
@@ -226,8 +226,8 @@ or an executable in PATH."
   :type 'string
   :group 'go)
 
-(defcustom godoc-and-godef-command "godoc"
-  "Which executable to use for `godoc' in `godoc-and-godef-command'.
+(defcustom godoc-and-godef-command "go doc"
+  "Which executable to use for `godoc' in `godoc-and-godef'.
 Must be 'godoc' and not 'go doc' and can be an absolute path or
```

Comment:
This doc string doesn't seem right anymore.

Reply:
Fixed.